packages/cozy-mespapiers-lib/src/components/Contexts/helpers.js

```diff
@@ -0,0 +1,40 @@
+import { index, addDocs } from '../../utils/search'
+import { FILES_DOCTYPE } from '../../doctypes'
+
+const updateFile = doc => {
+  if (doc.trashed) {
+    index.remove(doc._id)
+  }
```

Comment:
If a doc is changed, e.g. file rename, it won't be taken into account?

Reply:
effectivement tout comme pour les contacts, il faut qu'on gère la modification d'un files via le realtime 👍